firehose.tf

```diff
@@ -1,4 +1,4 @@
-resource "aws_cloudwatch_log_group" "group" {
+resource "aws_cloudwatch_log_group" "firehose" {
```

Comment:
Changing the attribute name will cause the log group to be recreated.  Not sure it's worth the risk given the attribute name is not exposed externally.

Reply:
done